src/shell/command_line.cpp

```diff
@@ -374,3 +374,74 @@ void CommandLine::Shift(unsigned int amount)
 		}
 	}
 }
+
+bool CommandLine::FindBoolArgument(const std::string& name, bool remove,
+                                   char short_letter)
+{
+	const std::string double_dash = "--" + name;
+	const std::string dash        = '-' + name;
+	char short_name[3]            = {};
+	short_name[0]                 = '-';
+	short_name[1]                 = short_letter;
+	return FindExist(double_dash.c_str(), remove) ||
+	       FindExist(dash.c_str(), remove) ||
+	       (short_letter && FindExist(short_name, true));
+}
+
+std::string CommandLine::StringArgHelper(const char* name)
```

Comment:
How does it help exactly? 😄 Can you try to make the name reflect what the function is actually doing? "Helper", "Util", etc. can be literally anything.

Reply:
Renamed to `FindRemoveSingleString`